nbformat/tests/base.py

```diff
@@ -7,13 +7,13 @@
 
 import os
 import unittest
-
+import io
 
 class TestsBase(unittest.TestCase):
     """Base tests class."""
 
-    def fopen(self, f, mode=u'r'):
-        return open(os.path.join(self._get_files_path(), f), mode)
+    def fopen(self, f, mode=u'r',encoding='utf-8'):
+        return io.open(os.path.join(self._get_files_path(), f), mode, encoding=encoding)
```

Comment:
Since this is a test utility and the tests pass, I don't think we ever open notebook files in binary mode. This should be fine, and we can add a conditional if we need to use this for binary files in the future.